test_sets/smoke_tests_non_media.txt

```diff
@@ -0,0 +1,9 @@
+./test_scripts/Smoke/Policies/001_PTU_all_flows.lua
```

Comment:
@GetmanetsIrina Propose to add these test scripts to existing smoke test set (not a new one)